module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -441,6 +441,11 @@ public function getHolding($bibId, array $patron = null, array $options = [])
             };
             for ($j = 0; $j < count($itemBody->items); $j++) {
                 $item = $itemBody->items[$j];
+                $supressed = isset($item->discoverySuppress)
+                    ? $item->discoverySuppress : null;
+                if ($supressed == true) {
```

Comment:
How about simplifying the whole block to:

<pre>
if ($item->discoverySuppress ?? false) {
    continue;
}
</pre>

Oh, how I love the null coalescing operator. ;-)

Reply:
Thanks! much better.